lms/static/sass/views/_shoppingcart.scss

```diff
@@ -637,390 +517,464 @@
   }
 
   .discount {
-      border-bottom: 2px solid $light-gray1;
-      border-top: 2px solid $light-gray1;
-      margin: $baseline 0;
-      padding: 17px $baseline 15px;
-      min-height: 45px;
-
-      .code-text {
-
-        a {
-          color: $blue1;
-          font-size: 18px;
-          text-transform: lowercase;
-          font-weight: 600;
-          display: inline-block;
-          padding: ($baseline/2) 0;
-          cursor: pointer;
-        }
+    margin: $baseline 0;
+    border-top: $discount-border;
+    border-bottom: $discount-border;
+    padding: ($baseline*0.85) ($baseline) ($baseline*0.75) ($baseline);
+    min-height: ($baseline*2.25);
 
-        span{
-          display: inline-block;
-          padding: 9px 0;
-          margin-right: -($baseline);
-
-          b{
-            font-weight: 600;
-            font-size: 24px;
-            padding-left: $baseline;
-            letter-spacing: 0;
-          }
+    .code-text {
```

Comment:
Add @include clearfix() here so that the discount code error text isn't on top of the border.